content/kubermatic/master/operation/etcd/etcd-launcher/_index.en.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "[Experimental] Etcd Launcher"
```

Comment:
Can we consistently call it `etcd` and not `Etcd`?

Reply:
I am using `Etcd launcher` as a sort of a title on the topic and `etcd-launcher` is used almost everything else.

Reply:
How do they say on the internet? "Thanks, but I hate it"? "Etcd launcher" to me is the worst of both worlds :-D Specifically the uppercase E, because I recently read https://cert-manager.io/docs/faq/style/ and have since been sensitivized (sic) to that kind of thing. ^^